.travis/script.sh

```diff
@@ -5,8 +5,7 @@ set -x
 if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
   mvn -Dstyle.color=always -B -U -P sign --settings travis-settings.xml deploy;
   # TODO - after testing this, make it only publish on a release build
-  # commented while I have a think.
-  # mvn site site:stage scm-publish:publish-scm;
+  mvn site site:stage scm-publish:publish-scm;
```

Comment:
I don't think Travis runs with `-e` by default - do you want to use `&&` to link these commands?

Reply:
a gotcha - `bash -e` ... ok done.